.github/workflows/conformance-report.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         with:
           gradle-version: 8.7
       - name: Download `conformance_test_results.ion` from target branch
-        uses: dawidd6/action-download-artifact@v6
```

Comment:
Hmm. Should keep the newer version otherwise may trigger another dependabot issue.